sources/lib/jam/tests/jam-test.dylan

```diff
@@ -11,23 +11,23 @@ define sideways method make-test-instance
   make(class)
 end method make-test-instance;
 
-define jam class-test <jam-state> ()
+define test test-<jam-state> ()
   //---*** Fill this in...
-end class-test <jam-state>;
+end test;
 
-define jam function-test jam-state-copy ()
+define test test-jam-state-copy ()
   //---*** Fill this in...
-end function-test jam-state-copy;
+end test;
 
-define jam function-test jam-variable ()
+define test test-jam-variable ()
   //---*** Fill this in...
-end function-test jam-variable;
+end test;
 
-define jam function-test jam-variable-setter ()
+define test test-jam-variable-setter ()
   //---*** Fill this in...
-end function-test jam-variable-setter;
+end test;
 
-define jam function-test jam-expand-arg ()
+define test test-jam-expand-arg ()
   with-test-unit ("jam-expand-arg simple variable expansion")
```

Comment:
Ideally this could be a suite and each of the `with-test-unit` blocks could be split into its own `test`.

Reply:
Yes, I left a TODO on the ones in the LLVM test suite. I'll go back at some point and get rid of `with-test-unit`.  (I'll file a bug so I don't forget.)